src/pages/dashboard/feedstock/[id].js

```diff
@@ -23,6 +23,12 @@ import { GoDatabase, GoTerminal } from 'react-icons/go'
 
 const Feedstock = () => {
   const router = useRouter()
+
+  const url =
+    process.env.NEXT_PUBLIC_VERCEL_ENV === 'production'
+      ? 'https://pangeo-forge.org'
+      : process.env.NEXT_PUBLIC_VERCEL_URL
```

Comment:
Ooh I just realized that this won't necessarily work locally -- I wonder if we can just use `window.location.origin` here.

Reply:
isn't it the case that we don't want a localhost URL for the opengraph metadata and falling back to the main site `pangeo-forge.org` suffices?

Reply:
That would work! I don't think that will happen currently because `process.env.NEXT_PUBLIC_VERCEL_ENV === 'production'` should be `false` locally.

I think we will want to keep using that check to ensure that the public-facing URL is used instead of the specific deploy URL from Vercel.

Maybe something like this would work:
```js
let url = 'https://pangeo-forge.org'
if (process.env.NEXT_PUBLIC_VERCEL_URL && process.env.NEXT_PUBLIC_VERCEL_ENV !== 'production') {
  url = process.env.NEXT_PUBLIC_VERCEL_URL
}
```